s3cloudfront/variables.tf

```diff
@@ -0,0 +1,19 @@
+provider "aws" {
+  region  = var.aws_region
+  version = "~> 2.70.0"
+}
+
+variable "aws_region" {
+  type = string
+  default = "us-east-1"
+}
+
+variable "fp_context" {
+  type = string
+}
+variable "domain" {
+  type = string
+}
+variable "aws_account_id" {
```

Comment:
It doesn't look like we are using the account ID anywhere in the Terraform code for the CDN so maybe we could remove this, and also all references to it elsewhere in the code.